### ISSUE TITLE
sts needs to be added to INSTALLED_APPS in global_settings

### ISSUE THREAD

**Issue by leipzig** [chop-dbhi/varify-data-warehouse#34]
sts tables are not being created with migrate

**Comment by naegelyd** [comment]
What are the steps to recreate this error?

-Don

**Comment by leipzig** [comment]
```
(vdw-env)[leipzigj@raboso varify-data-warehouse]$ ./bin/manage.py samples queue all_varify
DEBUG Scanning source directory: all_varify
DEBUG opening all_varify/all.snpeff.vcf in load_samples
DEBUG Trying to create NA12878 sample record
DEBUG NA12878 created
DEBUG Opening all_varify/MANIFEST in vdw.samples.models load_content
ERROR error processing manifest all_varify/MANIFEST
Traceback (most recent call last):
  File "./vdw/samples/management/subcommands/queue.py", line 57, in _queue
    load_dict = load_samples(manifest_path, database)
  File "./vdw/samples/pipeline/handlers.py", line 136, in load_samples
    sts.transition(sample, 'Sample Manifest Created')
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/sts/shortcuts.py", line 4, in transition
    return System.get(obj).transition(*args, **kwargs)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/sts/models.py", line 167, in get
    obj = cls.objects.get(content_type=ct, object_id=obj.pk)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/manager.py", line 131, in get
    return self.get_query_set().get(*args, **kwargs)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/query.py", line 361, in get
    num = len(clone)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/query.py", line 85, in __len__
    self._result_cache = list(self.iterator())
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/query.py", line 291, in iterator
    for row in compiler.results_iter():
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 763, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 818, in execute_sql
    cursor.execute(sql, params)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/backends/util.py", line 40, in execute
    return self.cursor.execute(sql, params)
  File "/nas/is1/leipzig/vdw-env/lib/python2.7/site-packages/django/db/backends/postgresql_psycopg2/base.py", line 52, in execute
    return self.cursor.execute(query, args)
DatabaseError: relation "sts_system" does not exist
LINE 1: ...s_system"."created", "sts_system"."modified" FROM "sts_syste...
```

adding sts to INSTALLED_APPS, followed by migrate fixes the error

**Comment by naegelyd** [comment]
@leipzig Thanks for the report. This was fixed in 8170ee41ce5f4. New release will be out soon.
